### PR TITLE
add pull after checkout command

### PR DESCRIPTION
**Pull request by andrewandante**
Should rectify the problem where only the 3 branch updates. `git pull --all` doesn't do what it says on the tin - is closer to `git fetch --all && git pull`.

### CODE REVIEW

**Comment by dhensby** [comment]
Have you been able to test this to see if it does fix the issue?

**Comment by andrewandante** [comment]
Tested locally and confirmed that it does fix the issue.

File: src/Console/CheckoutCommand.php [hunk 1]
```diff
@@ -32,8 +32,17 @@ public function run(InputInterface $input, OutputInterface $output)
             if (file_exists($root . '/.git')) {
                 $output->writeln("Updating <info>$name</info>");
                 $repo = RepoFactory::repoFor($root, $output);
-                $repo->run('remote', ['update']);
-                $repo->run('pull', ['--all']);
+                $repo->run('fetch', ['--all']);
+                foreach ($config['versions'] as $branch => $alias) {
+                    $trueBranch = $branch;
+                    if (array_key_exists('versionmap', $data)) {
+                        $trueBranch = $config['versionmaps'][$data['versionmap']][$branch];
```
Comment (dhensby) [inline src/Console/CheckoutCommand.php]: you might want to add an `array_key_exists` for this.

File: src/Console/CheckoutCommand.php [hunk 2]
```diff
@@ -32,8 +32,17 @@ public function run(InputInterface $input, OutputInterface $output)
...
+                    }
+                    if ($trueBranch ) {
```
Comment (dhensby) [inline src/Console/CheckoutCommand.php]: white space

**Comment by tractorcow** [comment]
Looks good to me. You happy with this @dhensby ?

**Comment by dhensby** [comment]
next on @andrewandante is to get tests passing :)

**Comment by chillu** [comment]
Note that this isn't deployed yet, I'm just following up with @robbieaverill :)

**Pull request merged**